django_project/feti/management/commands/scraping_campus.py

```diff
@@ -124,6 +124,7 @@ def scrape_campus(div):
     prov = create_provider(provider)
```

Comment:
let's change these to proper names. 
Maybe prov should rather be provider and provider should rather be provider_name?